apps/web/src/views/PoolDetail/components/MyPositions.tsx

```diff
@@ -155,10 +177,16 @@ export const MyPositions: React.FC<{ poolInfo: PoolInfo }> = ({ poolInfo }) => {
                 filter={filter}
                 setCount={setCount}
                 setTotalLiquidityUSD={setTotalLiquidityUSD}
+                setHandleHarvestAll={setHandleHarvestAll}
               />
             ) : null}
             {['v2', 'stable'].includes(poolInfo.protocol) ? (
-              <MyV2OrStablePositions poolInfo={poolInfo} setCount={setCount} setTotalTvlUsd={setTotalLiquidityUSD} />
+              <MyV2OrStablePositions
+                poolInfo={poolInfo}
+                setCount={setCount}
+                setTotalTvlUsd={setTotalLiquidityUSD}
+                setHandleHarvestAll={setHandleHarvestAll}
```

Comment:
better to use a props like `isHarvestAll` to control the child method, avoid the child change back to parent's state

Reply:
what `isHarvestAll` for? this is actually setState on purpose

Reply:
the child set a callback(_handleHarvestAll) back to the parent, it's not a best practice.
i means you can pass a `isHarvestAll` to the child, and the child do the `handleHarvestAll` for itself, avoid to setstate back to the parent